### ISSUE TITLE
Workflow should be assigned to admin sets

### ISSUE THREAD

**Issue by scholarworks** [projecthydra/sufia#2844]
### Summary

Workflows are currently linked to and generated by work types. In order to allow the most flexibility in workflow assignment, this should happen at the admin set level.
### Expected behavior
1. Define a workflow with the following syntax

```
{
  "workflows": [
    {
      "name": "default",
      "label": "Default Workflow",
      "description": "A single submission step, default workflow",
      "actions": [{
        "name": "deposit",
        "from_states": [{"names": ["new"], "roles": ["depositing"]}],
        "transition_to": "deposited",
        "notifications": [{
          "notification_type": "email",
          "name": "confirmation_of_submitted_to_ulra_committee",
          "to": ["creating_user"],
          "cc": ["advising"]
        }],
        "methods": [
          "CurationConcerns::Workflow::ActivateObject"
        ]
      }]
    }
  ]
}
```
1. In the admin dashboard, assign the workflow to a selected admin set.
2. When items are submitted to the admin set, the workflow is applied.
### Actual behavior

[IN PROGRESS]

**Comment by mjgiarlo** [comment]
@scholarworks The json above is now supported in CC.

**Comment by jcoyne** [comment]
This will require a database change, probably to "permission_templates" to add a `workflow_id` column

**Comment by scholarworks** [comment]
The "create_persmission_template" db files are in the permission_template branch. Is that work that needs to be merged with mediated_depist_epic?

**Comment by jcoyne** [comment]
@scholarworks Yeah, we need someone (like you) to review an merge that other PR.

**Comment by scholarworks** [comment]
@jcoyne - Cool. I'll start working on that and hit you up for any q's.

**Comment by mjgiarlo** [comment]
@scholarworks should we close this ticket now?

**Comment by mjgiarlo** [comment]
I believe this should have been closed by #2877. Closing. Reopen if I got that wrong.
